src/markdown-pages/docs/managers.md

```diff
@@ -21,7 +21,7 @@ title: "Managers"
-| **[z/OS Batch z/OS MF Manager](/docs/managers/zos-batch-zos-mf-manager)**<br> ![beta](../../images/beta.svg)     | Provides the default implementation of the z/OS Batch Manager using z/OS MF. Can only be used via the z/OS Batch Manager interface.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| **[z/OS Batch Manager](/docs/managers/zos-batch-zos-mf-manager)**<br> ![beta](../../images/beta.svg)             | Provides the default implementation of the z/OS Batch Manager using z/OS MF. Can only be used via the z/OS Batch Manager interface.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
 | **[z/OS Console z/OS MF Manager](/docs/managers/zos-console-zos-mf-manager)**<br> ![beta](../../images/beta.svg) | Provides the default implementation of the z/OS Console using z/OS MF. Can only be used via the z/OS Console Manager interface.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
can the same be applied to the z/OS Console and z/OS File Manager please?

Reply:
Done.